packages/uniforms/src/SimpleSchemaBridge.js

```diff
@@ -1,11 +1,10 @@
 /* global Package */
 
+import Bridge         from './Bridge';
 import cloneDeep from 'lodash.clonedeep';
+import filterDOMProps from './filterDOMProps';
 import invariant from 'fbjs/lib/invariant';
-
-import Bridge         from './Bridge';
 import joinName       from './joinName';
-import filterDOMProps from './filterDOMProps';
```

Comment:
Why you've changed it? At the top, we list all npm packages, then local dependencies.